ui/templates/upload.html

```diff
@@ -1,5 +1,6 @@
-{% extends "base.html" %}
+{% extends "repo_base.html" %}
```

Comment:
I think we still want it to extend repo_base, because when we add taxonomy to the navigation, we probably want it there.  We also probably want to let them know what repo they are in:
![Uploading image.png…]()
It also gets a bit hard to slice and dice, but I'll pull that out.